doc/release_notes.rst

```diff
@@ -31,6 +31,12 @@ This release contains bug fixes and extensions to the features for optimization
 
 * When solving ``n.lopf(pyomo=False, store_basis=True, solver_name="cplex")`` an error raised by trying to store a non-existing basis is caught.
 
+* Added track iterations of global constraints
+
+* overwrite ``p_nom`` and ``s_nom`` of line and link attributes to original values after final optimisation
+
+* When solving ``n.lopf(pyomo=False)``, PyPSA now constrains the dispatch variables for non extendable components with actual constraints, not with standard variable bounds. This allows retrieving shadow prices for all dispatch variables when running ``n.lopf(pyomo=False, keep_shadowprices=True).   
+
```

Comment:
There was a new release on Wednesday and the release notes for your pull request should go above this release (v0.17.1). Also "When solving..." is now duplicated in the release notes.